geopolars/src/geoseries.rs

```diff
@@ -87,6 +98,15 @@ pub trait GeoSeries {
     /// implicitly closed by copying the first tuple to the last index.
     fn is_ring(&self) -> Result<Series>;
 
+    /// Returns a GeoSeries with each of teh geometries roatered by a fixed x and y ammount arround
+    /// some origin.
+
+    fn rotate(&self, angle: f64, origin: TransformOrigin) -> Result<Series>;
```

Comment:
It would be nice to include in the docstring whether `angle` is expected to be in degrees/radians.